### PR TITLE
Improve flow of dev quick start

### PR DESCRIPTION
**Pull request by markusjura**
Improves the developer quick start flow based on the feedback from Kevin and Stefan during the hackathon. Also updates this documentation page to the newest plugin versions and necessary sbt keys.

### CODE REVIEW

File: src/main/play-doc/developer/DevQuickStart.md [hunk 1]
```diff
@@ -16,21 +16,12 @@ The focus of this section is to get started quickly. The full documentation of t
 * [[Deploying bundles|DeployingBundles]]
 * [[ConductR sandbox cluster|ConductrSandbox]]
 
-## Vendor lock-in
```
Comment (markusjura) [inline src/main/play-doc/developer/DevQuickStart.md:19]: Removed the `Vendor lock-in` chapter. It shouldn't be part of a quick start. In the previous section we are talking that this page focuses to get started quickly. Afterwards the next section `Vendor lock-in` talks about non-quick-start related topics which breaks the reading flow.

@huntc: Should we keep it removed or should we put this information to another chapter?

Comment (huntc) [inline src/main/play-doc/developer/DevQuickStart.md:19]: I think it'd be nice to retain somewhere... but I can't suggest where.

**Comment by huntc** [comment]
@rocketpages Can you please review?

File: src/main/play-doc/developer/DevQuickStart.md (hunk 1 above)
Comment (markusjura) [inline src/main/play-doc/developer/DevQuickStart.md:19]: I can create a new contribution page

**Comment by markusjura** [comment]
PR for 1.0 branch is #102.

**Pull request merged**